src/App.tsx

```diff
@@ -1,10 +1,73 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+
+export enum FilterType {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
+
+function getPreparedTodos(todos: Todo[], filter: FilterType) {
+  let preparedTodos = [...todos];
+
+  if (filter) {
```

Comment:
can we get rid of let?
we can return immediately and pass switch case in callback